.github/workflows/tests.yml

```
@@ -196,7 +196,7 @@ jobs:
       PYTEST_MARKER: ${{ matrix.test-type == 'unit' && 'not integration' || 'integration' }}
       PYTEST_SPLITS: ${{ matrix.test-type == 'unit' && '2' || '3' }}
       REQUIREMENTS_TRUSTSTORE: ${{ contains('3.10|3.11|3.12', matrix.python-version) && '--file tests/requirements-truststore.txt' || '' }}
-      CONDA_TEST_SOLVERS: ${{ github.event_name == 'pull_request' && 'libmamba' || 'libmamba,classic' }}
+      CONDA_TEST_SOLVERS: libmamba,classic
```

Maybe something like the following, or remove this change all together until #14264 is decided?
```suggestion
      CONDA_TEST_SOLVERS: ${{ matrix.default-channel == 'conda-forge' && 'libmamba' || 'libmamba,classic' }}
```

---

@zklaus Could you work on #14264? I've left a note there

---

#14264/#14270 has been included in this PR in https://github.com/conda/conda/pull/14267/commits/fadb8ec4ff6db10b995f3cff46de1a708ed2c85b since that change cannot be merged without the fixes here